app/shared/src/__tests__/__snapshots__/PdfViewer.test.js.snap

```diff
@@ -0,0 +1,39 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`handles errors 1`] = `
```

Comment:
This test is weird - it doesn't really test `handles errors` case, right?

Reply:
You are right, it works as a visual test when rendered in the playground, but as a snapshot it is weird, is it better to remove it?

Reply:
I would prefer to remove it or really test the `onError` callback.